dd-sdk-android/src/main/kotlin/com/datadog/android/core/internal/system/BroadcastReceiverSystemInfoProvider.kt

```diff
@@ -72,9 +74,15 @@ internal class BroadcastReceiverSystemInfoProvider :
         val level = intent.getIntExtra(BatteryManager.EXTRA_LEVEL, -1)
         val scale = intent.getIntExtra(BatteryManager.EXTRA_SCALE, 100)
 
+        val resolvedBatteryStatus = SystemInfo.BatteryStatus.fromAndroidStatus(status)
+        val resolvedBatteryLevel = (level * 100) / scale
+        sdkLogger.d(
```

Comment:
won't it be too noisy?

Reply:
Exactly what @xgouchet said also. I removed it ;)